src/index.js

```diff
@@ -65,14 +67,18 @@ module.exports = {
           let pragma, name, mocks, skipSelection
 
           // Load invocation mocks
-          if (existsSync(jsonMocks)) {
+          /**/ if (existsSync(jsonMocks)) {
```

Comment:
Is the comment needed?
```suggestion
          if (existsSync(jsonMocks)) {
```

Reply:
Needed? Nah, just a dumb little thing I do to align a series of `if` / `else if` conditions. 😅